packages/shared/src/lib/config.ts

```diff
@@ -7,11 +7,10 @@ export const graphqlUrl = `${apiUrl}/graphql`;
 
 export const fallbackImages = {
   avatar:
-    'https://res.cloudinary.com/daily-now/image/upload/s--O0TOmw4y--/f_auto/v1715772965/public/noProfile',
+    'https://res.cloudinary.com/daily-now/image/upload/s--qsFuKGv_--/t_logo/public/noProfile.jpg',
```

Comment:
Ideally we don't use this specified URL though, now it will force `.jpg` where previously it would format auto.

Reply:
Updated and removed the .jpg extension!